app/uk/gov/hmrc/transitmovements/repositories/DeparturesRepository.scala

```diff
@@ -50,11 +55,19 @@ import scala.util.Success
 import scala.util.Try
 import scala.util.control.NonFatal
 
+case class GetDepartureMessageIdsDTO(result: NonEmptyList[MessageId])
```

Comment:
Are these in the best place?

Reply:
I can see why it's in this file (it is private to it), but perhaps it should be a nested class to enforce the hierarchy, rather than here.

Reply:
Ah, okay. I see why it isn't an inner class.

Note that this is added as a codec to the repository in its constructor. Because of this, this _cannot_ be an inner class because you end up with a chicken and egg scenario - and Scala doesn't like it.

Thinking ahead, this class will likely be used in he arrivals repository too. So, I'll move this into its own file.